examples/cross_build/toolchain_packages/toolchain_macos_linux_cross/README.md

```diff
@@ -0,0 +1,53 @@
+# MacOs to Linux cross-compilation example
+
+## Introduction
+
+This example is a recipe that enables cross-compilation from *MacOs* to *Linux*. The example is limited to cross-compiling 
```

Comment:
I would also make here a more accurate description, "enable" is too vague, this is about creating and using a toolchain package for cross-building from macOS to Linux.